### ISSUE TITLE
Error on test cases

### ISSUE THREAD

**Issue by janesuede** [JuiceMind/JuiceMind#644]
Test cases spinning for a long time and then nothing happens or you get error  (these exact same test cases were working before)

**Comment by karlr-dev** [comment]
hi @mrlo-codes , you're having this error when testing the solution? (when you run the tests from the solution tab that's what it does is run the tests on the solution code). Even with the new testing system?

**Comment by mrlo-codes** [comment]
> hi [@mrlo-codes](https://github.com/mrlo-codes) , you're having this error when testing the solution? (when you run the tests from the solution tab that's what it does is run the tests on the solution code). Even with the new testing system?

Yes, here's the result of one example test run under the solutions tab:

<img width="575" alt="Image" src="https://github.com/user-attachments/assets/6fb4d6b5-d75b-46ff-a071-c7c8bfa61e97" />
